src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -190,10 +191,20 @@ class HttpGoogleDataprocDAO(appName: String,
   private def getClusterConfig(machineConfig: MachineConfig, initScript: GcsPath, clusterServiceAccount: Option[WorkbenchEmail], credentialsFileName: Option[String], stagingBucket: GcsBucketName): DataprocClusterConfig = {
     // Create a GceClusterConfig, which has the common config settings for resources of Google Compute Engine cluster instances,
     // applicable to all instances in the cluster.
-    // Set the network tag, network, and subnet. This allows the creaed GCE instances to be exposed by Leo's firewall rule.
-    val gceClusterConfig = new GceClusterConfig()
-      .setTags(List(defaultNetworkTag.value).asJava)
-      .setSubnetworkUri(defaultVPCSubnet.value)
+    // Set the network tag, network, and subnet. This allows the created GCE instances to be exposed by Leo's firewall rule.
+    val gceClusterConfig = {
```

Comment:
+1 
This is exactly what I would have preferred to do in my last PR but lacked the scala-foo to express coherently.